content/02.introduction.md

```diff
@@ -157,14 +157,50 @@ This excessive clotting in lung capillaries has been suggested to related to a d
 Individual outcomes have varied significantly, and reported hospitalization rates have been wide-ranging based on age and location, with various underlying health conditions being disproportionately reported [@doi:10.15585/mmwr.mm6915e3].
 Outcomes in patients that are hospitalized have generally been poor, with rates of admission to the intensive care unit (ICU) upwards of 15% in both Wuhan, China and Italy [@doi:10.1056/nejmoa2002032; @doi:10.1001/jama.2020.2648; @doi:10.1001/jama.2020.4031].
 
-Though older populations are generally considered the most vulnerable, pediatric infections are also a concern.
-[@doi:10.1001/jamapediatrs.2020.1467] reviewed studies on children infected with COVID-19 during the early months of COVID-19 in China.
-Most of the children reviewed had mild symptoms such as fever, dry cough, fatigue, nasal congestion and/or runny nose or were asymptomatic.
-Gastrointestinal symptoms such as vomiting or diarrhea were occasionally reported.
-Radiologic findings were not always mentioned, but when mentioned they included bronchial thickening, ground-glass opacities, and/or inflammatory lesions.
-The results of COVID-19 infection can vary greatly among pediatric patients as well as adults.
-In particular, some children have experienced a severe inflammatory syndrome after COVID-19 infection, similar in some respects to Kawasaki disease or to Kawasaki disease shock syndrome [@doi:10.1093/jpids/piaa069;@doi:10.1001/jama.2020.10369; @doi:10.1016/j.ajem.2020.05.117].
-Heart failure can also occur [@doi:10.1161/CIRCULATIONAHA.120.048360].
+#### Subpopulations of Special Concern  
+
+In the context of the United States, persons diagnosed with COVID-19 are more likely to require hospitalization if they are of male sex, of older age, and/or of Black/African American background [@doi:10.1377/hlthaff.2020.00598; @doi:10.15585/mmwr.mm6925e1].
```

Comment:
I don't see our earlier comments/conversation about this. When I browsed the articles you cited & the 14 state CDC article cited in the second article, then I also saw it written something like:
"Hospitalized COVID-19 patients are more commonly older, male, of black race, and have underlying conditions."
or similarly
"Reports of U.S. patients hospitalized with SARS-CoV-2 infection (the virus that causes COVID-19) describe high proportions of older, male, and black persons (2-4)." followed by another sentence about underlying conditions
[quotations from summary & intro from second reference]
I think it makes sense from the beginning to cover both types of major observations seen so far -- things having to do with probability of getting infected and things having to do with probability that infection is severe. You do this later on, but I think your opening sentence should also include both in order to get readers to see both key interesting points from the beginning. If you want just one sentence then I think something closer to the quotation above is better because it covers both aspects and then it can be broken up into components later on; the other option is putting both your current sentence & something like quotation above to suggest multiple components from beginning. (Though everyone has worked on this, still going back to this discussion b/c it's at beginning of section)

Reply:
Thank you, @yemarshall!  I guess the ideal thing to do would be to tell which demographic, lifestyle, or medical variables contribute to each stage of the process, like:
1. Risk factors for infection:...
2. Risk factors for hospitalization given infection:...
3. Risk factors for death given hospitalization:...
Then we could talk about, for example, are the elderly more likely to get infected than the young, or do they just get sicker if and when they do get infected?  
But I'm not sure whether we can really do that with the literature we have now, which is preliminary and doesn't have prospective longitudinal data on the general population.  Studies on hospitalized samples can provide information about #3 but only #3.  Studies using vital statistics can probably provide information on a muddled mixture of 1, 2, and 3, but probably only under assumptions that aren't safe to make (equal and honest reporting by the authorities in all localities). 
We certainly could talk about these general issues first, and then mention the specific disparities and risk factors.  But it might be kind of embarrassing for us in a way, because we will have to say "Here are some important questions!  And the answers are ... we don't know."  So I thought of putting the theoretical issues at the end of the subsection instead, like the limitations section at the end of a paper.  I guess that doesn't really solve the problem either -- we can only say "Here's a bunch of findings!  By the way, it would be nice if we could explain them."  @rando2, what do you think?

Reply:
@dziakj1 would these perhaps be questions worth raising in the discussion? #514 
This is a huge area of research and definitely we are not going to be able to answer these questions. I made the discussion file specifically to support brainstorming on this topic because it seems like there are far more questions than answers and a lot of these concerns may be easier to address as outstanding concerns in the discussion!

Reply:
You're right, @rando2.  I know that the deadline is looming ever closer!  Would you like to just reorganize this section however you see fit?  I will approve whatever changes you make.

Reply:
Hi @dziakj1 I'm happy to! I'm a lot sharper in the morning so I'll work on this first thing tomorrow.